c4709881.lua

```diff
@@ -65,7 +77,7 @@ function c4709881.sptg1(e,tp,eg,ep,ev,re,r,rp,chk,chkc)
 	if chk==0 then return zone~=0
 		and (Duel.IsExistingTarget(c4709881.spfilter1,tp,LOCATION_GRAVE,0,1,cc,e,tp,zone)
 		or (e:GetHandler():GetLinkedGroupCount()>=2 and Duel.GetLocationCount(tp,LOCATION_MZONE)>-1
-			and Duel.IsExistingTarget(c4709881.spfilter0,tp,LOCATION_GRAVE,0,1,cc,e,tp))) end
+			and Duel.IsExistingTarget(c4709881.spfilter1,tp,LOCATION_GRAVE,0,1,cc,e,tp))) end
```

Comment:
[Actually, that's the point.](https://github.com/Fluorohydride/ygopro-scripts/pull/904/files/50fbe9bc4320234caab5b0ec0b5cf18c41b97967#diff-ceff6cb74416bbf7f088d80a5f63fa93R56) I kind of simplified the code, as well ^^'